pom.xml

```diff
@@ -84,8 +84,7 @@
         <guice.version>4.1.0</guice.version>
         <jetty.version>9.4.10.v20180503</jetty.version>
         <jersey.version>1.19.3</jersey.version>
-        <!-- jackson 2.7.x causes injection error and 2.8.x can't be used because avatica is using 2.6.3 -->
```

Comment:
We're already at Avatica `1.12.0`, which is at Jackson 2.9.x: https://mvnrepository.com/artifact/org.apache.calcite.avatica/avatica-core/1.12.0